src/blosc2/ndarray.py

```diff
@@ -385,7 +385,7 @@ def _check_allowed_dtypes(
     ):
         raise RuntimeError(
             "Expected LazyExpr, NDArray, NDField, C2Array, Proxy, np.ndarray or scalar instances"
-            f" and you provided a '{type(value)}' instance"
+            + f" and you provided a '{type(value)}' instance"
```

Comment:
You don't need to add the `+`, as this is implicit when two strings are in different lines.